src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -211,7 +211,7 @@ public sealed class NotNullIfNotNullAttribute : Attribute
         protected const string IsExternalInitTypeDefinition = @"
 namespace System.Runtime.CompilerServices
 {
-    public sealed class IsExternalInit
+    public static class IsExternalInit
```

Comment:
Thanks @RikkiGibson for adding the [BCL API](https://github.com/dotnet/runtime/pull/37763). I see that you've changed the type to `static` there, to match other similar compiler types. So I'm updating in our tests to match.